src/components/atoms/PasswordInput/PasswordInput.tsx

```diff
@@ -0,0 +1,42 @@
+import {TextInput, ImageBackground, Pressable} from 'react-native';
+import {styles} from './styles';
+import {useState} from 'react';
+import IconEyeOn from '../../../static/icons/IconEyeOn.svg';
+import IconEyeOff from '../../../static/icons/IconEyeOff.svg';
+import {colors} from '../../../static/consts/colors';
+
+type TPasswordInputProps = {
+  value: string;
+  onChange: (e: string | React.ChangeEvent<any>) => void;
+  onBlur: (e: any) => void;
+  additionalStyles?: any;
+};
+
+export const PasswordInput = ({
+  value,
+  onChange,
+  onBlur,
+  additionalStyles,
+}: TPasswordInputProps) => {
+  const [hidden, setHidden] = useState(true);
+  return (
+    <ImageBackground
+      source={require('../../../static/imgs/inputBackground.png')}
+      resizeMode="cover"
+      style={additionalStyles}
+    >
+      <Pressable onPress={() => setHidden(prev => !prev)} style={styles.icon}>
+        {hidden ? <IconEyeOff /> : <IconEyeOn />}
+      </Pressable>
+      <TextInput
+        style={styles.input}
+        placeholder={hidden ? '******' : 'Hasło'}
```

Comment:
Jest coś takiego jak **secureTextOnly**, to od razu ci zadziała jak chcesz, bez jakichś kurwa on pressów i placeholderów

Reply:
tbh nigdzie czegoś takiego nie widzę i google też mi nic o tym nie mówi

Reply:
nic takiego nie widziałem w docsach, google też o czymś takim nie mówiło

Reply:
https://reactnative.dev/docs/textinput
secureTextEntry, sry xd

Reply:
no to niżej mam to użyte chyba że mówisz o tym żeby w placeholderze zostawić tylko "hasło"

Reply:
no to niżej mam to użyte chyba że mówisz o tym żeby w placeholderze zostawić tylko "hasło"

Reply:
W placeholderze zostaw po prostu "****", nie ma sensu jakas podmiana placeholdera